README.md

```diff
@@ -262,22 +119,24 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
```

Comment:
Just as the previous reviewer mentioned, please remove any trace of a template that is not personalized from your ReadMe. This comment was intended as a guide for you on what to add to this section but it has no relevance to other users interacting with your project. That is why it is good to always delete lines like this in your ReadMes.